src/main.py

```diff
@@ -5,9 +5,9 @@
 from task3 import decorator_3
 from task4 import decorator_4
 
+test_decorator = decorator_4
```

Comment:
Adding a new variable for the tested decorator to avoid changing it for every function at every test